pools/custom_pools_test.go

```diff
@@ -0,0 +1,20 @@
+package pools
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	customValidatorFile = "pubkeys.json"
+)
+
+func TestReadCustomValidator(t *testing.T) {
+	keys, err := ReadCustomValidatorsFile(customValidatorFile)
+	require.Equal(t, nil, err)
+	require.Equal(t, 10, len(keys))
```

Comment:
i think its also important to assert that each key was parsed correctly. testing just the size is not enough :)